src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
+  const filteredBySex = people.filter(elem => elem['sex'] === 'm');
+
+  const filteredByCentury = filteredBySex.filter(elem =>
+    (Math.ceil(elem['died'] / 100) === century));
+
+  const sumOfMenAgesWithCent = filteredByCentury.reduce((acc, b) =>
+    acc + (b['died'] - b['born']), 0);
```

Comment:
```suggestion
  const sumOfMenAgesWithCent = filteredByCentury.reduce((acc, b) => (
     acc + (b['died'] - b['born']
   )), 0);
```
pay attention to the formatting